examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -80,6 +82,19 @@ impl FrontendNode {
             oak::channel_close(wh.handle).unwrap();
             oak::channel_close(rh.handle).unwrap();
         }
+
+        {
+            // Create a new node to handle HTTP requests
+            info!("Starting HTTP server pseudo-node on port {}.", HTTP_ADDR);
+            let http_channel =
+                oak::http::init(HTTP_ADDR).expect("could not create HTTP server pseudo-Node!");
+            oak::node_create(
+                &oak::node_config::wasm(FRONTEND_MODULE_NAME, "http_oak_main"),
```

Comment:
How does it choose where to send incoming invocations (gRPC compared to HTTP) ?

Reply:
HTTP and gRPC servers are listening on separate ports. The `http_channel` here is the `invocation_receiver`, and is used to start an event-loop for listening to incoming HTTP requests in the `http_oak_main` node defined below. The gRPC server is created in the main node (`frontend_oak_main`) and has its own event-loop for incoming gRPC requests. 

This bit of the code for creating `http_oak_main` could be moved to `frontend_oak_main`, but I thought here is a better place for it, since the rest of the additional node and channel creation logic is here. 

I think we could use the same pattern to allow an Oak application to have both gRPC and HTTP server nodes ([Ref discussion on Slack](https://project-oak.slack.com/archives/CHE9E13C3/p1601305095011300)).